Branch-SDK/Branch-SDK/Branch.m

```diff
@@ -216,8 +215,8 @@ + (BranchActivityItemProvider *)getBranchActivityItemWithParams:(NSDictionary *)
 
 #pragma mark - Configuration methods
 
-+ (void)setDebug {
-    [BNCPreferenceHelper setDevDebug];
+- (void)setDebug {
```

Comment:
Because `setDebug` is a non-permanent method (for testing purposes only), I figured we could just directly remove it. Thoughts?